src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -345,15 +350,15 @@ If you want to keep your source maps secret and choose not to upload your source
   content=__alert_content
 %}
 
-## Context
+## Adding Context Data
```

Comment:
Data is implied. "Adding Context" or "Adding Contextual Data" but I prefer the former